ui/app/styles/utils/_size_variables.scss

```diff
@@ -3,18 +3,23 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
+@function rem($value, $base: 16) {
+  @return calc($value / $base) + 0rem;
+}
+
 /* General sizing in rem values used largely for text sizing.*/
-$size-1: 3rem; // 48px, same as $spacing-xxl
-$size-2: 2.5rem; // 40px
-$size-3: calc(24 / 14) + 0rem; // ~1.714rem ~27px
-$size-4: 1.5rem; // 24px same as $spacing-l
-$size-5: 1.25rem; // 20px
-$size-6: 1rem; // 16px same as $spacing-m
-$size-7: calc(13 / 14) + 0rem; // ~.929rem ~15px
-$size-8: calc(12 / 14) + 0rem; // ~.857rem  ~13.7px
-$size-9: 0.75rem; // 12px same as $spacing-s
-$size-10: 0.5rem; // 8px same as $spacing-xs
-$size-11: 0.25rem; // 4px same as spacing-xxs
+$size-1: rem(42); // 42px or 2.63rem
+$size-2: rem(36); // 35px or 2.19rem
```

Comment:
Should the px comment be updated here or is the value passed to `rem()` incorrect?

Reply:
nice catch! the rem is incorrect. I will get that fixed!